libmozevent/phabricator.py

```diff
@@ -91,8 +92,9 @@ def update_state(self, build):
             return
 
         # Check this build has been awaited between tries
+        exp_backoff = (2 ** (self.max_retries - retries_left + 1) - 1) * self.sleep
```

Comment:
Nit: you could do `(2 ** (self.max_retries - retries_left)) * self.sleep` and set max_retries to 5 to have more or less the same effect but simpler code.